packages/p2p/src/utils/advertiser.ts

```diff
@@ -0,0 +1,24 @@
+import { localize } from 'Components/i18next';
+import { isMobile } from '@deriv/shared';
+
+/**
+ * Function to get the message to be displayed in the blocked users modal
+ * @function getMessage
+ * @param {number} user_blocked_count // the numbers of users who have blocked the current user
+ * @return { string }
+ * */
+
+export const getMessage = (user_blocked_count: number): string => {
```

Comment:
```suggestion
export const getMessage = (user_blocked_count: number) => {
```

Do we need the return type?